backend/src/main/java/com/javahelp/backend/accountdeletion/DeleteManager.java

```diff
@@ -0,0 +1,38 @@
+package com.javahelp.backend.accountdeletion;
+
+import com.javahelp.backend.data.IUserStore;
+import com.javahelp.model.user.User;
+import com.javahelp.model.user.UserPassword;
+
+/**
+ * A class for managing an account deletion action.
+ */
+class DeleteManager {
```

Comment:
We should be consistent with our use case class naming conventions, this is called manager, I just pushed a PR that used interactor instead. The CRC cards say manager, so it might make sense to use that and rename any interactors to be consistent, but we might also want to have this discussion with the whole team present, just to keep everyone up to date and get everyone's input!